fighters/samusd/src/acmd/specials.rs

```diff
@@ -188,12 +188,44 @@ unsafe fn game_specialairlw(fighter: &mut L2CAgentBase) {
     }
 }
 
+#[acmd_script( agent = "samusd_cshot", script = "sound_Shoot", category = ACMD_SOUND, low_priority)]
```

Comment:
script name should be in lowercase: `sound_shoot`

Reply:
Will fix when i get home!